profiler/test/Datadog.Profiler.Native.Tests/LinuxStackFramesCollectorTest.cpp

```diff
@@ -290,7 +293,11 @@ class LinuxStackFramesCollectorFixture : public ::testing::Test
 TEST_F(LinuxStackFramesCollectorFixture, CheckSamplingThreadCollectCallStack)
 {
     auto* signalManager = GetSignalManager();
-    auto collector = LinuxStackFramesCollector(signalManager);
+
+    auto [configuration, mockConfiguration] = CreateConfiguration();
+    EXPECT_CALL(mockConfiguration, UseBacktrace2()).WillOnce(Return(true));
```

Comment:
Would it be possible (like in C#) to have the same test to run once with backtrace2 and once with the previous implementation?
That way, both implementations would be tested

Reply:
oh good one.
I was adding test for the configuration but forgot this one.